<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyWidget_OrphansBase</class>
 <widget class="QFrame" name="PropertyWidget_OrphansBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
   <item row="0" column="0">
    <widget class="QLabel" name="keepLabelStart">
     <property name="toolTip">
      <string>Ensure that first lines of a paragraph won't end up separated from the rest (known as widow/orphan control)</string>
     </property>
     <property name="text">
      <string>Don't separate first:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ScrSpinBox" name="keepLinesStart">
     <property name="toolTip">
      <string>Ensure that first lines of a paragraph won't end up separated from the rest (known as widow/orphan control)</string>
     </property>
     <property name="suffix">
      <string> lines</string>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="keepLabelEnd">
     <property name="text">
      <string>Don't separate last:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ScrSpinBox" name="keepLinesEnd">
     <property name="toolTip">
      <string>Ensure that last lines of a paragraph won't end up separated from the rest (known as widow/orphan control)</string>
     </property>
     <property name="suffix">
      <string> lines</string>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="keepTogether">
     <property name="toolTip">
      <string>If checked, ensures that the paragraph won't be split across multiple pages or columns</string>
     </property>
     <property name="text">
      <string>Do not split paragraph</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="keepWithNext">
     <property name="toolTip">
      <string>If checked, automatically moves the paragraph to the next column or page if the next paragraph isn't on the same page or column</string>
     </property>
     <property name="text">
      <string>Keep with next paragraph</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
